VirtoCommerce.Platform.Web/Scripts/app/security/blades/account-list.js

```diff
@@ -46,7 +46,12 @@ function ($scope, accounts, dialogService, uiGridHelper, bladeNavigationService,
                 if (remove) {
                     bladeNavigationService.closeChildrenBlades(blade, function () {
                         var itemIds = _.pluck(selection, 'userName');
-                        accounts.remove({ names: itemIds }, blade.refresh);
+                        accounts.remove({ names: itemIds }, function () {
```

Comment:
I don't think that is good idea to allow user remove self account from system, could we prevent this action?

Reply:
Yes, we could do this. But in many systems removing self account is a legal action (like Github). Maybe better to add confirmation that user is trying to delete himself (or add that info to initial confirmation on Delete action)?

Reply:
Yes, confirmation will be a good choice.

Reply:
Decided to restrict possibility of self deletion on server side.